SimpleAkkaDotNetCluster/SimpleCluster/SimpleCluster.csproj

```diff
@@ -7,12 +7,6 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Akka" Version="1.4.38" />
-    <PackageReference Include="Akka.Cluster" Version="1.4.38" />
-    <PackageReference Include="Akka.Cluster.Tools" Version="1.4.38" />
```

Comment:
We'll be pulling all of our dependency from the shared library so that all projects will use the same reference NuGet packages